src/components/VCarousel/VCarousel.vue

```diff
@@ -71,6 +62,11 @@
 
         !this.isBooted && this.cycle && this.restartInterval()
         this.isBooted = true
+
+        this.$emit('input', this.current)
+      },
+      value () {
```

Comment:
After talking to @jacekkarczmarczyk, I think I'm fine with keeping current, as it allows one to use the carousel without declaring a v-model. What do you think @johnleider?